appengine/standard_python37/bigquery/templates/timeout.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+{#
```

Comment:
HTML comment syntax

Reply:
This is Jinja2 template syntax so that the license header doesn't appear in the rendered result.

Reply:
Got it. Thanks.